mungegithub/Makefile

```diff
@@ -15,6 +15,7 @@ KUBECONFIG ?= $(HOME)/.kube/config
 KUBEDIR ?= $(GOPATH)/src/k8s.io/kubernetes
 CACHEDIR ?= /tmp/mungegithub-cache
 WWW ?= $(PWD)/$(APP)/www
+CONFIGMAP ?= k8s-kubernetes
```

Comment:
nit: CONFIGMAP is a confusing name, I think. It selects not just the config map, but the entire target of the submit queue. How about "SQTARGET"? or "TARGETREPO"? or "TARGETCONFIG"? or just "TARGET"?